pkg/render/logstorage.go

```diff
@@ -584,28 +580,12 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 				},
 			},
 			VolumeMounts: []corev1.VolumeMount{
-				// Even though it won't be used for TLS, we need to mount the secret volume for initialization.
```

Comment:
Comment to the reviewer. In earlier versions of ECK 3 volumes were mounted by ECK to the already present init container. When cert management is enabled we wanna add a fourth. ECK does something that looks like a strategic merge, but if we did not specify the existing 3, they would be left out and the container would crash.

In the new ECK version these 3 mounts are no longer present, so there is no need any longer to add them in our code.